.github/workflows/flake8.yml

```diff
@@ -0,0 +1,16 @@
+name: Flake8
+on: pull_request
+jobs:
+  flake8:
+    name: Check code with Flake8
+    runs-on: ubuntu-20.04
+    container: fedora:32
+    steps:
+      - name: Install Pipenv and Git
+        run: dnf install -y pipenv git
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup environment
+        run: pipenv sync --dev
+      - name: Run Flake8
+        run: pipenv run flake8 --max-line-length 120
```

Comment:
I think it's more acceptable to have 4 spaces for indent although 2 should also work. @ifireball @dmaizel wdyt?

Reply:
Yep, IMO 2 is fine

Reply:
For YAML 2 usually works better because of the list syntax. Consider:
```
aaa:
    bbb:
        mmm: 5
        nnn: 6
    ccc: 2
ddd:
    - 1234
    - 5678
```
V.s.:
```
aaa:
  bbb:
    mmm: 5
    nnn: 6
  ccc: 2
ddd:
  - 1234
  - 5678
```